<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VaRA-TS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../icons/straus64.png</normaloff>../../../icons/straus64.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuConfig">
     <property name="title">
      <string>Config</string>
     </property>
     <addaction name="actionCreate_BenchBuild_Config"/>
     <addaction name="actionSave_Config"/>
    </widget>
    <addaction name="actionVaRA_Setup"/>
    <addaction name="actionInteractionFilter_Editor"/>
    <addaction name="menuConfig"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionExampleView"/>
    <addaction name="actionCR_BarView"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExampleView">
   <property name="text">
    <string>ExampleView</string>
   </property>
  </action>
  <action name="actionCR_BarView">
   <property name="text">
    <string>CR-BarView</string>
   </property>
  </action>
  <action name="actionSetup">
   <property name="text">
    <string>Setup</string>
   </property>
  </action>
  <action name="actionDownload_VaRA">
   <property name="text">
    <string>Initialize VaRA</string>
   </property>
   <property name="toolTip">
    <string>Downloads and builds VaRA</string>
   </property>
  </action>
  <action name="actionUpdate_VaRA">
   <property name="text">
    <string>Update VaRA</string>
   </property>
  </action>
  <action name="actionShow_VaRA_status">
   <property name="text">
    <string>Show VaRA status</string>
   </property>
  </action>
  <action name="actionVaRA_Setup">
   <property name="text">
    <string>VaRA Setup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save VaRA Config</string>
   </property>
  </action>
  <action name="actionCreate_BenchBuild_Config">
   <property name="text">
    <string>Create BenchBuild Config</string>
   </property>
  </action>
  <action name="actionInteractionFilter_Editor">
   <property name="text">
    <string>InteractionFilter Editor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
